static/js/biomedical/disease/data_processing_utils.ts

```diff
@@ -356,3 +358,20 @@ export function doesDiseasePrevalenceIDexist(data: GraphNodes): boolean {
   }
   return false;
 }
+/**
+ * Converts the disease parent data from a flat array to a tree array for tree visualization
```

Comment:
I don't think you're converting it to a tree array, seems like you're just converting it into a tree?

Reply:
Yes, changed it to only "tree"